index.html

```diff
@@ -16,8 +16,8 @@
   <meta property="og:image" content="https://hacktoberfestnh.github.io/img/hacktoberfestnh.png" />
 
   <!-- Favicons -->
-  <link href="img/favicon.png" rel="icon">
-  <link href="img/apple-touch-icon.png" rel="apple-touch-icon">
+  <!-- <link href="img/favicon.png" rel="icon"> -->
+  <!-- <link href="img/apple-touch-icon.png" rel="apple-touch-icon"> -->

```

Comment:
Why don't you delete these commented links?

Reply:
A lot of code is just commented to allow easier update later.
`favicon` should be changed once we have a good alternative.